### DISCUSSION TITLE
Voltar para faculdade ou focar em um curso ?

### DISCUSSION THREAD

**Discussion by silvaMatheus** [frontendbr/forum#654] [Discussão]
Eu estava cursando faculdade de Ciência da computação até o meio desse ano e tive q parar por motivos familiares, nesses meses q estava de ferias consegui um emprego até q bom (fora da area de tecnologia ) e me surgiu a ideia de mudar de faculdade pois a q eu estava fazendo (unicsul) nunca foi do meu agrado eu estava pensando em ir pra FIAP inicio do ano e fazer sistema de informação pois nesses 3 semestres de ciencia da computação só peguei matemática e outras materias q eu nao me interessei e eu quero mesmo trabalhar com a parte de Web(Front end e Back ) e aplicativos. Porem iria ter q começar do zero então me surgiu a ideia de no lugar de fazer uma facul focar em cursos como por exemplo esse novo da Caelum de formação FullStack (https://www.caelum.com.br/formacao-fullstack) gostaria de saber a opinião de vcs, seria mais interessante eu fazer SI na fiap ou fazer esse curso da caelum ou então fazer faculdade e esperar meio do ano e fazer esse curso ?

**Comment by Pedro-Souza** [comment]
Por que começar do zero?
Você pode cortar as matérias que você já fez em CC. Eu fiz 1 ano de CC mais aí tive que trocar de estado e aqui não tem CC na faculdade onde eu fazia e por alguns motivos acabei optando não continuar CC, resumindo, eliminei 6 matérias da grade.

**Comment by clucasalcantara** [comment]
Então cara, tudo depende muito do seu foco. Hoje para atuar na nossa profissão não é necessário um curso superior, apesar de que, na minha humilde opinião, na faculdade vc vai aprender justamente as matérias humanas,(que na real: precisamos sim ^^), que vc não vê nos cursos técnicos. 

Além disso, se vc tiver planos de ir morar no exterior e conseguir um job lá é MUITO mais difícil (não impossível, claro) conseguir um sponsor pra pagar seu visto.

*Eis o que eu fiz:*
Existem cursos hoje com duração de 2 anos e meio a 3, a distância, e que tem uma boa colocação no MEC e reconhecimento lá fora, hoje eu to cursando a Unicesumar, mas tem várias na mesma pegada, dá uma pesquisada!

**Comment by caioincau** [comment]
Eu não me formei, larguei no penúltimo ano pq precisei começar a trabalhar, hoje em dia "me arrependo" um pouco, pois algumas vagas precisam do canudo, como por exemplo a vaga para o CUBO, o @LFeh talvez tenha algo a acrescentar sobre isso.

**Comment by mahenrique94** [comment]
Apesar de achar que as faculdades não nos preparam para o mercado, é essencial ter um canudo, tanto para empregos no Brasil e indiscutível para fora do Brasil, porém a nivel de conhecimento acredito que com certeza a formação da Caleum vai lhe proporcionar muito mais.

**Comment by ericvvf** [comment]
Como falou o @clucasalcantara , **depende do teu foco.** Terminei o curso técnico pelo instituo federal, e logo me empreguei. Tive a oportunidade, neste meio tempo, de fazer o superior, porém, tive que optar, por motivos de horários, entre trampar e estudar. Escolhi trabalhar. Não digo que me arrependo, pois consegui muita experiência e consigo, hoje, me manter, por conta do trampo que escolhi. Porém, fiquei meio estagnado no tocante a poder concorrer a algumas vagas que surgiram no meio do caminho.

Hoje, estou fazendo novamente o superior, na mesma instituição que cursei o técnico, e acho improvável que eu faça o que fiz há 5 anos atrás.

Podes optar por um curso tecnológico, que dependendo da instituição, te prepara bem pro mercado, e você sai com o canudo superior, em menos tempo (geralmente um/dois anos a menos que licenciatura e bacharel)

Se você tiver tempo de fazer uma faculdade, e disposição, minha dica é essa: faça.

**Comment by andresantos-tech** [comment]
Comigo aconteceu praticamente a mesma coisa que com o @ericOnda . Fiz técnico num instituto federal e  comecei a fazer estágio no final do curso, e no começo do ano seguinte comecei uma faculdade (que, também por questões de horário, parei faltando cerca de 1 ano para concluir o curso). Eu fico um pouco arrependido por ter parado o curso, até porque fiz isso depois de já ter passado por 75% do "sofrimento". Se fosse hoje, eu tentaria me mudar para algum lugar próximo ao trabalho ou faculdade, nem que fosse só até terminar o curso.

O fim da história é que no começo desse ano voltei a fazer faculdade (tecnólogo, modalidade EAD, com duração de 2 anos e meio). Uma coisa que notei é que quanto mais o tempo passa mais difícil é fazer uma faculdade (porque as responsabilidades aumentam e o tempo tende a ser cada vez mais escasso).  O curso atual dá pra ir levando de boa por ser EAD, mas se fosse presencial seria bem complicado de fazer.

Em resumo: Faça faculdade. E quanto antes você começar mais tranquilo será. Cursos livres pra especialização você terá que fazer durante sua vida toda (ou pelo menos enquanto for desenvolvedor :p), então acho que uma coisa não compete com a outra.

**Comment by felipefialho** [comment]
Então, já tivemos algumas discussões muito boas sobre o tema: #39 #518 

De forma prática, em uma opnião MUITO pessoal, não acho que uma faculdade te torne melhor profissional. Não vai te dar base prática nenhuma e a base teórica pode ser conseguida estudando sozinho ou em cursos.

Mas existem outros pontos para considerar, uma faculdade facilita (ou pode facilitar):

- Vistos fora do país
- Vagas em alguns empregos que possuem essa exigência (algumas muito boas)

Entre outros beneficios que sabemos que um curso superior pode trazer.

Particulamente, gosto da estratégia de cursar técnologos, que trazem todas essas vantagens em apenas dois ou três anos. E mais tarde podem ser convertidos em pós-graduação ou até mestrado.

**Comment by matheusml** [comment]
Não existe resposta fácil pra isso, mas vou tentar expressar minha **opinião**.
A propósito, eu me formei em Ciência da Computação na UFF. 

### Resolver problemas difíceis
Durante os longos (e árduos) anos enquanto fazia faculdade, me perguntava qual era a utilidade de aprender matérias tão absurdas quanto Física I e II, Cálculo I, II e III, dentre outras que aparentemente não tinham nenhuma relação com a minha carreira.

Hoje, alguns anos após ter me formado, percebi que **a principal vantagem de estudar essas matérias era de me ensinar a resolver problemas difíceis**. E depois, quando cheguei no mercado, desenvolvimento _web_ era trivial comparado com os problemas que nós fomos obrigados a resolver.

### Networking
A [Concrete](https://www.concrete.com.br/) é a quinta empresa em que trabalho. Dessas cinco, somente uma das que trabalhei consegui o emprego sem ser por indicação. Todas as outras quatro foram por indicação direta de amigos.

Na faculdade você vai conhecer muita gente igual a você, que está passando pela mesma fase e procura os mesmos objetivos. Ou seja, um excelente lugar pra fazer o _Networking_ que vai te ajudar nas vagas de emprego no futuro.

### Vistos e vagas
E o terceiro e último ponto é o que o @LFeh citou acima. Pra algumas vagas ainda existe a obrigação de ter um ensino superior completo.

Espero ter ajudado.
Abraços

**Comment by Clayder** [comment]
Eu estou no último ano, no curso de ciência da computação em uma universidade federal.  Realmente a faculdade, não vai te dar base prática para entrar em um empresa, mas ela vai te dar uma excelente base teórica, em diversas áreas. 

Eu aprendi assuntos nas áreas de inteligência artificial, banco de dados, redes, games, programação, engenharia de software, sistema operacional  etc. 
Como meu foco sempre foi desenvolvimento web, então eu fui estudando e praticando por fora essa área, depois de um tempo eu comecei a pegar freelas. Atualmente, além de estagiar eu trabalho como freelancer.  

Não me arrependo de fazer faculdade, principalmente pelo motivo, que com certeza eu não teria estudado todas essas áreas sozinho e tem muitas vagas que exigem nível superior. 

 Mas isso tudo, depende do seu foco. Eu vejo que pelo fato de eu fazer faculdade, ainda mais no interior, eu demorei a entrar no mercado. Na cidade onde eu moro, é muito difícil conseguir emprego na área de desenvolvimento.

**Comment by lucianobarauna** [comment]
comcordo com o @LFeh . Costumo dizer que quem faz a faculdade é você e na maioria dos casos a vivência profissional vai te dar experiências muito boas. Na hora de solucionar um problema o conhecimento teórico que talvez..talvez você tem aprendido na faculdade o ajude solucioná-lo.
Enfim é faca de dois gumes, vai de você e sua necessidade

**Comment by leonardosouza** [comment]
@silvaMatheus se o seu interesse é "escapar" da matemática, sinto muito em informar mais este curso da FIAP que você citou tem bastante matemática, e logo no primeiro ano! :/ 

Eu vejo duas situações ai... um um ajuste de curso (talvez um técnico focado em web), caso você julgue muito importante para sua vida a faculdade, ou se está pensando em se posicionar no mercado rapidamente talvez um curso livre como o da Caelum faça mais sentido. 

Como citado aqui, a faculdade não é mandatória para atuar na área, mais pode te ajudar em muitas situações. Por fim, siga seu coração! :D

**Comment by MendesGabriel** [comment]
Só um relato:

Eu comecei em 2012 ADS na FATEC, fiz 1 ano e tive que parar para trabalhar.

Ano passado voltei para FATEC para fazer Sistemas para Internet (eliminei algumas matérias), consegui transferência externa para o Instituto Federal no curso de ADS agora no começo de 2017.

Esse semestre passei em engenharia da computação na univesp.br, curso totalmente EAD (alguns encontros presenciais a cada 15/30 dias). Como não é permitido por lei fazer 2 faculdades publicas ao mesmo tempo, abandonei o IF e vou fazer os 5 anos de engenharia. 

O que mais pesou na decisão foi a autonomia, vou assistir aula onde quiser, colocar o video em 2.0x, voltar quantas vezes quiser e como meu objetivo é morar fora do país um dia,  o titulo de engenheiro -ainda- conta muito. 

[ univesp.br, tem vários polos no estado de SP, a concorrência não é absurda e você não precisa ir todo dia depois do trampo assistir aula (muitas vezes inúteis) das 19 as 23h todo dia ]

**Comment by giioohbernini** [comment]
Muitas pessoas comentaram sobre fazer ou não faculdade, mas eu como um
aluno de SI da fiap indo para o ultimo ano, vou abordar mais sobre a
faculdade mesmo, pois dependendo da faculdade que você escolher, vai acabar
gastando muito dinheiro que poderia ser investido de outra forma.

Uma coisa posso te garantir sobre o curso de SI na FIAP, você pagará uma
mensalidade salgada só para ter um canudo com um pouco mais de relevância
por algumas empresas. Você realmente esta pagando uma mensalidade salgada
pelo marketing e não pelo valor de ensino.

Então caso você escolha voltar para faculdade, acredito que a FIAP não é o
melhor investimento. A ideia desses cursos de 2 a 3 anos de duração também
são muito validas, mas caso escolha terminar um bacharel de 4 anos de
duração, recomendo pesquisar outras instituições de ensino com uma
mensalidade mais barata!

Essa é uma visão que eu e a maioria dos alunos (da minha sala pelo menos)
compartilhamos. Espero ter ajudado para a sua decisão!

Em sex, 11 de ago de 2017 16:39, Leonardo Souza <notifications@github.com>
escreveu:

**Comment by silvaMatheus** [comment]
Bom pessoal demorei um pouco pra dar uma continuidade pois estava esperando juntar uma boa quantidade de respostas. Muito obg pelas respostas. dicas, relatos, algumas informações  q eu não sabia. 

Uma duvida q ficou, um tecnólogo possui o mesmo peso q um bacharel ? ou depende muito do caso? Se a resposta for sim qual o melhor  curso ? https://www.fiap.com.br/graduacao/#cursos  (obs: Eu foco muito na FIAP pois ela fica na rua de cima da minha casa kkkkkk 
 
Li muito sobre o peso de uma faculdade nessa área deste q criei esse tópico e vi muitas pessoas falando para eu colocar meus objetivos em um papel e ver se realmente era necessário um curso superior  então fiz isso. Meu objetivos:
Como eu ja falei no inicio 

>eu quero mesmo trabalhar com a parte de Web(Front end e Back ) e aplicativos.

**Objetivo principal** 

-Entrar  o mais rápido possível no mercado.

**Objetivos a longo prazo** 

-Daqui um tempo pegar um job remoto ou presencial fora do país.
-E sonhando um pouco alto daqui uns anos montar uma empresa de Sistemas para Internet 

Se um  Tecnólogo tiver o mesmo peso q um bacharel então seria mais vantajoso para esses meu objetivos certo ?

**Comment by isacjunior** [comment]
Eu optei pelo que foi dito pelo @LFeh. Fiz um curso tecnólogo em Sistemas para Internet que teve a duração de 2 anos. Acredito que a sua escolhe se baseia muito no que você pretende exercer no futuro. Um exemplo é a carreira acadêmica, se pretende isto, é essencial que se faça uma ótima graduação e posteriormente mestrado e doutorado. Mas se almeja a vida profissional como desenvolvedor, a graduação pode ser a porta de entrada para o mercado de trabalho.

**Comment by phhoffmann** [comment]
Alguém conhece cursos tecnólogos de sistemas pra internet em SP pra saber falar se é bom ou não? hehe Cursos EAD também valem! :)

**Comment by AmandaFonseca** [comment]
Eu tenho o técnico em Análise e desenvolvimento de sistemas, até onde eu sei para cargos públicos o candidato tem que ser bacharel, eu passei por maus bocados pq antes era da área da saúde e  minha faculdade não é lá essas coisas. Só acho o tecnologo interessante para quem já tem algum tipo de vivencia na área seja por meio de cursos ou profissionalmente falando.  No meu curso o tempo era curto para cobrir toda a grade curricular e os meus professores apresentavam um conteúdo muito superficial e pelos comentários dessa issue acho que isso acontece com quase todo mundo. Maasss mesmo gastando muito para aprender pouco eu recomendo que  você faça um curso superior sim pois o setor de Rh das empresas são péssimos para avaliarem os profissionais de T.I , diploma universitário acaba entrando como critério de seleção sim.
